assets/airports/rjss.json

```diff
@@ -0,0 +1,960 @@
+{
+    "airac": 2007,
+    "radio": {
+        "twr": "Sendai Tower",
+        "app": "Sendai Approach",
+        "dep": "Sendai Departure"
+    },
+    "icao": "RJSS",
+    "iata": "SDJ",
+    "magnetic_north": -8,
+    "ctr_radius": 55,
+    "ctr_ceiling": 16000,
+    "initial_alt": 16000,
+    "position": ["N38d08m23.00", "E140d55m01.00", "5.6ft"],
+    "rangeRings": {
+        "enabled": true,
+        "center": ["N38d08m23.00", "E140d55m01.00"],
+        "radius_nm": 5.0
+    },
+    "has_terrain": true,
+    "wind": {
+        "angle": 270,
+        "speed": 4
+    },
+    "arrivalRunway": "27",
+    "departureRunway": "27",
+    "defaultMaps": ["Base Map", "Extended Runway Centerline"],
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 160,
+            "airspace_class": "E",
```

Comment:
Not sure how Japanese aircraft is categorized, but this should be the closest American airspace class equivalent (most likely B or C)

Reply:
![image](https://user-images.githubusercontent.com/53888046/86028635-1eef7f80-ba00-11ea-9799-03dd674c2297.png)

This airspace classification is taken from the Civil Aviation Bureau AIP.

Reply:
Yeah so the problem is that every country has different airspace classifications and openScope is based on the American version. So we need to figure out what the American equivalent of Japanese class E airspace is.

Reply:
So the Japanese airspace classification is based on the ICAO classification ([ICAO Annex 11 Appendix 4](https://www.theairlinepilots.com/forumarchive/quickref/icao/annex11.pdf)); I'm not quite familiar with the FAA classification but what would class E be in the US?

ICAO adopted classifications

Class | Controlled | IFR | SVFR | VFR | ATC Clearance | Separation | Traffic information
-- | -- | -- | -- | -- | -- | -- | --
A | Controlled | Yes | No | No | Required | Provided for all flights | N/A
B | Controlled | Yes | Yes | Yes | Required | Provided for all flights | N/A
C | Controlled | Yes | Yes | Yes | Required | Provided for all IFR/SVFR to IFR/SVFR/VFR | Provided for all VFR
D | Controlled | Yes | Yes | Yes | Required | Provided for IFR/SVFR to other IFR/SVFR | Provided for all IFR and VFR
E | Controlled | Yes | Yes | Yes | Required for IFR and SVFR | Provided for IFR/SVFR to other IFR/SVFR | Provided for all IFR and VFR flights where possible
F | Uncontrolled | Yes | No | Yes | Advisory only | Provided for IFR/SVFR to other IFR/SVFR where possible | Provided where possible if requested
G | Uncontrolled | Yes | No | Yes | Not provided | Not provided | Provided where possible if requested

Reply:
I _think_ class C would be the closest American equivalent. @erikquinn Could you confirm this?